### DISCUSSION TITLE
Proxy - [SSL: CERTIFICATE_VERIFY_FAILED]

### DISCUSSION THREAD

**Discussion by Filch-Project** [kevinzg/facebook-scraper#682] [Q&A]
I am using paid residential proxies and I am getting an IP back from the "set_proxy" function.  However, once I get into the get_posts function (for groups) I'm getting SSL_CERTIFERCATE_FAILED.  I'm trying to figure out where I should set requests to "verify=False" and if that would solve my problem.  Any thoughts or suggestions?

**Comment by neon-ninja** [comment]
Try https://github.com/kevinzg/facebook-scraper/commit/d1aa4692a4d04a5cf82956053c71845673d310e1

**Reply by Filch-Project** [reply]
Setting both to True did not work.  Setting them both to False did get me past the SSL verification.  Now i have a new error.  I haven't had a chance to dig into it yet.  Possibly facebook is rejection the ip?  Below is the traceback:

  File "/home/adam/.local/lib/python3.8/site-packages/facebook_scraper/facebook_scraper.py", line 1023, in _generic_get_posts
    for i, page in zip(counter, iter_pages_fn()):
  File "/home/adam/.local/lib/python3.8/site-packages/facebook_scraper/page_iterators.py", line 92, in generic_iter_pages
    response = request_fn(next_url)
  File "/home/adam/.local/lib/python3.8/site-packages/facebook_scraper/facebook_scraper.py", line 830, in get
    raise exceptions.UnexpectedResponse("Your request couldn't be processed")
facebook_scraper.exceptions.UnexpectedResponse: Your request couldn't be processed

**Reply by neon-ninja** [reply]
I think you misunderstood - this commit makes it possible to pass your desired setting through from `set_proxy`. Like, `set_proxy("http://localhost:9050", verify=False)`.

That error is pretty generic - is it reproducible? Can you post the code to reproduce the error?

**Reply by brandon-scott-pritchard** [reply]
My first time trying to use proxies here (and just in general). I have user & pass for each proxy, but I'm not sure how to pass it into set_proxy. Is there a way to do that?
